pages/api/v1/bounty/curator/submissions/editSubmission.ts

```
@@ -0,0 +1,95 @@
+
+		const { title, content, tags, link, reqAmount, proposerAddress, parentBountyIndex } = req.body;
+		if (
+			!title?.length ||
+			!content?.length ||
+			!reqAmount ||
+			new BN(reqAmount || 0).eq(ZERO_BN) ||
+			!proposerAddress?.length ||
+			!getEncodedAddress(proposerAddress, network) ||
+			(link?.length && !(link as string)?.startsWith('https:')) ||
+			(tags?.length && !!tags?.some((tag: string) => typeof tag !== 'string')) ||
+			isNaN(parentBountyIndex)
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)